src/app/auth-page/auth-page.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
When creating components via cli you can pass the flag --spec=false, it will tell cli to omit redundant (for now) test files